pkg/service/service.go

```diff
@@ -221,26 +220,16 @@ func PushKubernetesResources(client kclient.ClientInterface, k8sComponents []dev
 		}
 	}
 
-	madeChange := false
-
 	// create an object on the kubernetes cluster for all the Kubernetes Inlined components
 	for _, c := range k8sComponents {
 		u, er := GetK8sComponentAsUnstructured(c.Kubernetes, context, devfilefs.DefaultFs{})
 		if er != nil {
 			return er
 		}
 
-		isOperatorBackedService, er := PushKubernetesResource(client, u, labels)
```

Comment:
Whey are you removing this? User can always add Kubernetes resources manually to the devfile

Reply:
Mistake. Rushing to get this one done. 😞